tools/erc-repository-indexer/erc-contract-indexer/erc-registry/mainnet/erc-721.json

```diff
@@ -0,0 +1,110 @@
+[
+  {
+    "contractId": "0.0.1778110",
+    "address": "0x00000000000000000000000000000000001b21be",
+    "name": "Genesis SpheraHead",
+    "symbol": "SPH"
+  },
+  {
+    "contractId": "0.0.2015187",
+    "address": "0x00000000000000000000000000000000001ebfd3",
+    "name": "UserNftMinter",
+    "symbol": "UMT"
```

Comment:
Q: are we going to add serial ID next or?

Reply:
Right serial number is a tricky part. In the context of ERC token contracts, the serial number is equivalent to the token ID, and a single token can have multiple token IDs (or serial numbers, if you will). The `name` and `symbol` fields are retrieved via contract calls to the Mirror Node APIs using the standard sighashes for the `name()` and `symbol()` methods. However, I believe there is no standard API to retrieve all token IDs associated with a token.  

Looking at [[this interface from OpenZeppelin](https://docs.openzeppelin.com/contracts/5.x/api/token/erc721#IERC721)](https://docs.openzeppelin.com/contracts/5.x/api/token/erc721#IERC721), the token ID appears to be a predefined variable typically known beforehand and used as a function argument for operations that require it, rather than a retrievable value.  

Even if there's such an API to retrieve all IDs existed, I believe it would be better for downstream applications like MNE to handle this task to avoid outdated information. Just thinking out loud but MNE could address this by making on-demand requests like when a user clicks on a token, the application could fetch a list of owners and their associated token IDs or provide similar functionality. This approach would somehow align with how Etherscan manages on their [token balances](https://etherscan.io/token/0x524cab2ec69124574082676e6f654a18df49a048#balances) pages.

Reply:
Sounds good.
This and the empty fields comment before make me think it'd be good to add a section that talks to known issues that consumers of the file will have to watch out for.
Something to add to the README here or in a follow up PR